pandas/io/tests/test_gbq.py

```diff
@@ -707,10 +707,63 @@ def test_invalid_option_for_sql_dialect(self):
                          private_key=_get_private_key_path())
 
         # Test that a correct option for dialect succeeds
-        # to make sure ValueError was due to invalid dialect
+        # to make sure ValueError was due to invalid  dialect
         gbq.read_gbq(sql_statement, project_id=_get_project_id(),
                      dialect='standard', private_key=_get_private_key_path())
 
+    def test_query_with_parameters(self):
+        sql_statement = "SELECT @param1 + @param2 as VALID_RESULT"
+        config = {
+            'query': {
+                "useLegacySql": False,
+                "parameterMode": "named",
+                "queryParameters": [
+                    {
+                        "name": "param1",
+                        "parameterType": {
+                            "type": "INTEGER"
+                        },
+                        "parameterValue": {
+                            "value": 1
+                        }
+                    },
+                    {
+                        "name": "param2",
+                        "parameterType": {
+                            "type": "INTEGER"
+                        },
+                        "parameterValue": {
+                            "value": 2
+                        }
+                    }
+                ]
+            }
+        }
+        # Test that an invalid query without query config
+        with tm.assertRaises(ValueError):
```

Comment:
why is this necessary? I thought ``configuration`` is an optional paramter? when is it needed